src/rgw/rgw_file.cc

```diff
@@ -1290,7 +1290,7 @@ int rgw_getattr(struct rgw_fs *rgw_fs,
     rc = -(fs->getattr(rgw_fh, st));
   }
 
-  return -(fs->getattr(rgw_fh, st));
```

Comment:
I can't remember what I was playing with here, but I think we want just this line (return -(expr)), and to zap the two calls that assign rc.  What do you think?